__tests__/app.test.js

```diff
@@ -130,6 +130,29 @@ describe('/api/articles/:article_id', () => {
                 });
         });
     })
+    describe('GET article by id should now include comment_count property', () => {
+        test('When given an article_id, returned object includes a comment_count property', () => {
+            const testArticleResult = {
+                article_id: 3,
+                title: 'Eight pug gifs that remind me of mitch',
+                topic: 'mitch',
+                author: 'icellusedkars',
+                body: 'some gifs',
+                created_at: '2020-11-03T09:12:00.000Z',
+                votes: 0,
+                article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+                comment_count: 2
+            }
+            return request(app)
+                .get('/api/articles/3')
+                .expect(200)
+                .then(({ body }) => {
+                    const { article } = body
+                    expect(article.comment_count).toBe(2)
```

Comment:
if it matches the object above, we are already checking that comment_count is 2 - i think you'll be fine with having either one or the other expect statement here for this test but you dont need both :)